docs/oscar_references.bib

```diff
@@ -1789,14 +1728,16 @@ @Article{Tay87
   volume        = {3},
   pages         = {76--85},
   year          = {1987},
-  url           = {https://arxiv.org/abs/2201.09155}
+  eprint        = {2201.09155},
+  primaryclass  = {math.GR}
```

Comment:
As far as I can tell, the `primaryclass` is only printed for a very bib entries, like this one; for most other arxiv ones it is not. Presumably it is *only* printed for those using the `eprint` field? I guess this is an artifact of the "hack" used to abuse `@article` for use in eprints. I wonder why they are not simply implement `@Misc` suitably?

Reply:
If we want to have them visible for arxiv only papers as well, it needs to go into the journal field. I can adapt that tomorrow

Reply:
It is not important from my POV, I was just at first confused why it appeared for some papers but not others.

so, don't waste your time on this :-)